packages/block-editor/src/components/block-variation-picker/index.native.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import {
+	ScrollView,
+	View,
+	Text,
+	TouchableWithoutFeedback,
+	Platform,
+} from 'react-native';
+import { map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { withSelect, useDispatch } from '@wordpress/data';
+import { compose, usePreferredColorSchemeStyle } from '@wordpress/compose';
+import { createBlock } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import {
+	PanelBody,
+	BottomSheet,
+	UnsupportedFooterControl,
+} from '@wordpress/components';
+import { Icon, close } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import MenuItem from '../inserter/menu-item';
+import styles from './style.scss';
+
+function BlockVariationPicker( { isVisible, onClose, clientId, variations } ) {
+	const createBlocksFromInnerBlocksTemplate = ( innerBlocksTemplate ) => {
```

Comment:
This seems like a function that could be outside of the component :) We don't need to declare a new one on each render.